docs/framework/data/wcf/accessing-data-service-resources-wcf-data-services.md

```diff
@@ -70,26 +70,28 @@ http://services.odata.org/Northwind/Northwind.svc/Orders?$filter=not endswith(Sh
-|`$inlinecount`|Requests that a count of the number of entities returned in the feed be included with the feed. For more information, see [OData: Inlinecount System Query Option ($inlinecount)](https://go.microsoft.com/fwlink/?LinkId=186975).|  
-  
+|`$orderby`|Defines a default sort order for entities in the returned feed. The following query orders the returned customers feed by county and city:<br /><br /> `http://services.odata.org/Northwind/Northwind.svc/Customers?$orderby=Country,City`<br /><br />|  
+|`$top`|Specifies the number of entities to include in the returned feed. The following example skips the first 10 customers and then returns the next 10:<br /><br /> `http://services.odata.org/Northwind/Northwind.svc/Customers?$skip=10&$top=10`<br /><br />|  
+|`$skip`|Specifies the number of entities to skip before starting to return entities in the feed. The following example skips the first 10 customers and then returns the next 10:<br /><br /> `http://services.odata.org/Northwind/Northwind.svc/Customers?$skip=10&$top=10`<br /><br />|  
+|`$filter`|Defines an expression that filters the entities returned in the feed based on specific criteria. This query option supports a set of logical comparison operators, arithmetic operators, and predefined query functions that are used to evaluate the filter expression. The following example returns all orders the postal codes of which do not end in 100:<br /><br /> `http://services.odata.org/Northwind/Northwind.svc/Orders?$filter=not endswith(ShipPostalCode,'100')`<br /><br />|  
+|`$expand`|Specifies which related entities are returned by the query. Related entities are included as either a feed or an entry inline with the entity returned by the query. The following example returns the order for the customer 'ALFKI' along with the item details for each order:<br /><br /> `http://services.odata.org/Northwind/Northwind.svc/Customers('ALFKI')/Orders?$expand=Order_Details`<br /><br />|  
+|`$select`|Specifies a projection that defines the properties of the entity are returned in the projection. By default, all properties of an entity are returned in a feed. The following query returns only three properties of the `Customer` entity:<br /><br /> `http://services.odata.org/Northwind/Northwind.svc/Customers?$select=CustomerID,CompanyName,City`<br /><br />|  
+|`$inlinecount`|Requests that a count of the number of entities returned in the feed be included with the feed.|  
+  
+For more information, see `4. Query String Options` at [OData: URI Conventions](https://www.odata.org/documentation/odata-version-2-0/uri-conventions/).
```

Comment:
```suggestion
For more information, see section `4.9. Inlinecount System Query Option ($inlinecount)` at [OData: URI Conventions](https://www.odata.org/documentation/odata-version-2-0/uri-conventions/).
```

Reply:
This is meant to include information about the whole table. not just `$inlinecount` specifically. so I think it should remain `4. Query String Options` and just add `section` after `see`, @mairaw .

Reply:
Yes, I just noticed that and made a change.